implementations/absinthe-federation/lib/products_web/schema.ex

```diff
@@ -51,8 +51,30 @@ defmodule ProductsWeb.Schema do
       external()
     end
 
+    field :average_products_created_per_year, :integer do
+      requires_fields("totalProductsCreated yearsOfEmployment")
+
+      resolve(fn
+        %{
+          email: "support@apollographql.com",
+          total_products_created: total_products_created,
+          years_of_employment: years_of_employment
+        } = user,
+        _args,
+        _ctx ->
+          {:ok, trunc(total_products_created / years_of_employment)}
```

Comment:
Had to truncate this decimal value since the schema says its an `Int` and absinthe doesn't allow you to encode a decimal as an integer